src/seedu/addressbook/AddressBook.java

```diff
@@ -1147,14 +1104,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix(p/, d/, etc) from parameter string
```

Comment:
notice that the comment still uses `p/, d/` etc., related to address book.
the exercise was to make this method as generic as possible. the same has to be reflected in the comments as well